compiler/test/suites/stdlib.re

```diff
@@ -14,72 +14,174 @@ describe("stdlib", ({test, testSkip}) => {
   assertSnapshot("stdlib_cons", "[1, 2, 3]");
   assertSnapshot(
     "stdlib_equal_1",
-    "import * from \"list\"; (1, 2) is (1, 2)",
+    {|
+      import * from "list"
+      (1, 2) is (1, 2)
+    |},
   );
   assertSnapshot(
     "stdlib_equal_2",
-    "import * from \"pervasives\"; (1, 2) == (1, 2)",
+    {|
+      import * from "pervasives"
+      (1, 2) == (1, 2)
+    |},
   );
   assertSnapshot(
     "stdlib_equal_3",
-    "import * from \"list\"; [1, 2, 3] == [1, 2, 3]",
+    {|
+      import * from "list"
+      [1, 2, 3] == [1, 2, 3]
+    |},
+  );
+  assertSnapshot(
+    "stdlib_equal_4",
+    {|
+      import * from "list"
```

Comment:
@ospencer why does this and every test below it import list?

Reply:
I believe the original intent was to test that imports work, but we have lots of tests for that now.